### PR TITLE
Add springboot docker

### PR DESCRIPTION
**Pull request by AntonZhdanov**

### CODE REVIEW

**Review by IvanFadieiev — APPROVED** [review]
LGTM!)

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+
+services:
+  book-store:
```
Comment (nacenik) [inline docker-compose.yml:15]: I'd recommend you change the order. Better to start services and then service that depends on others.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+RUN java -Djarmode=layertools -jar application.jar extract
+
+# Final stage
+FROM openjdk:17-jdk-slim
+WORKDIR application
+COPY --from=builder application/dependencies/ ./
+COPY --from=builder application/spring-boot-loader/ ./
+COPY --from=builder application/snapshot-dependencies/ ./
+COPY --from=builder application/application/ ./
+ENV JAVA_TOOL_OPTIONS="-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005"
```
Comment (nacenik) [inline Dockerfile]: I'd also recommend you move this to the docker-compose instead of dockerfile

**Review by nacenik — CHANGES_REQUESTED** [review]
GJ, let's improve it :)

**Review by boroda4436 — APPROVED** [review]
Good job!

**Pull request merged**